### ISSUE TITLE
implement BP entry range validation

### ISSUE THREAD

**Issue by mattStorer** [OHSUCMP/coach#80]
propose the following limits:

systolic : 50 - 300
diastolic: 40 - 200

8/22 COACH notes. These were the numbers suggested by Richelle. Recheck with Dave.

**Comment by mattStorer** [comment]
I think we also wanted to have some sort of immediate "your BP is too high" alert if they enter a BP where systolic >= 180 or diastolic >= 120, but I'm not sure just how we want that to look.  it was mentioned during Tuesday morning's COACH meeting.  what should that text look like?  what icon, if any, should be displayed?

**Comment by ghumman96** [comment]
CAn not enter BP- Hard stop limits;
systolic : 50 - 300
diastolic: 30 - 200

**Comment by MichelleBobo** [comment]
@mattStorer Are you still needing more information on this, or have your questions been answered? If the path forward is clear, can we remove the needs info tag?

**Comment by mattStorer** [comment]
> @mattStorer Are you still needing more information on this, or have your questions been answered? If the path forward is clear, can we remove the needs info tag?

I think we can remove the tag.  this issue is at its core relates to hard-stop limits for entering systolic and diastolic values.  I believe the 'needs info' tag was added with respect to my comment on 8/22 about the 'your BP is too high' alert, but that is really a separate thing from the hard-stop limits where we prevent a user from entering values above or below a particular range.

we've discussed the need for, if someone enters systolic >= 180 or diastolic >= 120 that we enact a separate workflow, similar to the low-BP workflow recently discussed, that involves immediately returning to the home page and rerunning recommendations, which ought to trigger a pathway in the AE workflow that generates a card informing the user to take another reading.

but that's a separate thing.
